### PR TITLE
LESQ-757: Add aria-current to keystage selector

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

Music year: 1957

### Adds aria-current attribute to:
- Keystage selector buttons on Subject Listing
- Homepage tabs on Homepage
- Tier buttons on Unit Listing

## Issue(s)

Fixes #LESQ-757

## How to test

1. Go to https://deploy-preview-2477--oak-web-application.netlify.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
Screenshot 2024-06-06 at 08 43 33
Screenshot 2024-06-06 at 08 49 16

How it should now look:
Screenshot 2024-06-06 at 08 42 47
Screenshot 2024-06-06 at 08 48 43
Screenshot 2024-06-06 at 08 52 02

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "@mux/mux-node": "^8.5.1",
     "@mux/mux-player-react": "^2.6.0",
     "@oaknational/oak-components": "^0.38.0",
-    "@oaknational/oak-curriculum-schema": "^1.9.0",
+    "@oaknational/oak-curriculum-schema": "^1.14.0",
```
Comment (njblore) [inline package.json:75]: if you're updating the schema will need to add the pupil page fixes

Comment (k-huggs) [reply package.json:75]: Just doing your review atm, so will wait for it to go in first

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
Sorry there must have been some crossed wires. By unit filters I meant the threads on the right. The filter threads only add a query param so they shouldn't be page, instead true. The foundation/higher do navigate a user to a different page so should use page.
Screenshot 2024-06-07 at 08 39 46

**Review by njblore — APPROVED** [review]

**Review by jack-skerrett-oak — APPROVED** [review]
Works nicely now, sorry about the confusion :)

**Pull request merged**